src/fast_check/transform.rs

```diff
@@ -390,8 +493,48 @@ impl<'a> FastCheckTransformer<'a> {
         | Stmt::DoWhile(_)
         | Stmt::For(_)
         | Stmt::ForIn(_)
-        | Stmt::ForOf(_)
-        | Stmt::Expr(_) => Ok(false),
+        | Stmt::ForOf(_) => Ok(false),
+        Stmt::Expr(n) => match &mut *n.expr {
```

Comment:
```suggestion
        Stmt::Expr(Expr::Assign(assign_expr)) => {
```

And then `Stmt::Expr(_) => Ok(false)` for the rest.

Can you also move the `Ok(false)` branch to the end of the `match`?